scheduler/drilling-wits-depth-median/README.md

```diff
@@ -0,0 +1,38 @@
+# Drilling WITS Depth Median Backend App
+
+**Drilling WITS Depth Median** is a depth scheduled app 
+that calculates and stores `drilling.wits.depth` data __medians__.
+
+## Packaging
```

Comment:
It would be nice to reference our App Deployer script, or to use the standard Corva CLI, right?

Reply:
I was actually thinking about setting up Deployer for this app

Reply:
Will come back to this after API adds support for scheduled depth event. Because we can't deploy the app as for now.